.gitignore

```diff
@@ -38,3 +38,4 @@ pickle-email-*.html
 .env.local
 docker-compose.env
 docker-compose.yml
+production.rb
```

Comment:
Please add an empty line at the end of the file to make this red symbol disappear

Reply:
okay!